docs/api/classifier/index.rst

```diff
@@ -8,7 +8,7 @@ Classification models attempt to predict a target in a discrete space, that is a
 -  :doc:`classification_report`: A visual classification report that displays precision, recall, and F1 per-class as a heatmap.
 -  :doc:`confusion_matrix`: A heatmap view of the confusion matrix of pairs of classes in multi-class classification.
 -  :doc:`rocauc`: Graphs the receiver operating characteristics and area under the curve.
--  :doc:`class_balance`: Visual inspection of the target to show the support of each class to the final estimator.
+-  :doc:`../target/class_balance`: Visual inspection of the target to show the support of each class to the final estimator.
```

Comment:
Am still leaving the `ClassBalance` visualizer with the classification documentation because I know folks will want to reference it from there, and also because there isn't much documentation for the target module yet.